### ISSUE TITLE
Ghost lanes

### ISSUE THREAD

**Issue by originalfoo** [krzychu124/BrokenNodeDetector#6]
In several log files I've looked at recently I noticed users with this old mod subscribed:

https://steamcommunity.com/sharedfiles/filedetails/?id=536250255

It's listed in the 'other mod section of the official redit mods list:

https://www.reddit.com/r/CitiesSkylines/wiki/modlist

Would be a useful feature for broken nodes detector mod if phantom lanes are still a thing?

**Comment by krzychu124** [comment]
Yeah I thought about refactoring and implementing those features, but I suspended it because of no free time...
